src/Shiny.BluetoothLE.Common/Shiny.BluetoothLE.Common.csproj

```diff
@@ -4,8 +4,12 @@
         <VersionSuffix>beta</VersionSuffix>
         <Description>Shiny BluetoothLE - Common components for Hosting and Client</Description>
         <TargetFrameworks>netstandard2.0;xamarin.ios10;monoandroid90</TargetFrameworks>
-        <TargetFrameworks Condition=" $(OS) == 'WINDOWS_NT' ">uap10.0.16299;$(TargetFrameworks)</TargetFrameworks>
+        <TargetFrameworks Condition=" $(OS) == 'WINDOWS_NT' ">uap10.0.17763;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
Why is this being updated in all of the projects?  Is there API's that only become available as of this version?

Reply:
My installed sdk version is 17763. I'll install the 16299 test it and after that I'll make a new pull request.

Reply:
It works fine with version 16299